.gitignore

```diff
@@ -1,3 +1,3 @@
 importers/__pycache__
 tmp
-
+*.log
```

Comment:
You can also add:
`logs/*`
Since the code is dependent on a certain directory structure, it might be nice to add a logs dir to version control. You'll need to add an empty `logs/.gitkeep` file in order to do this.

Reply:
The log-relevant .gitignore commands are now:
```
*.log
logs/*.log
logs/*.json
```
I thought if I did just `logs/*` it would also ignore the `.gitkeep` right?

Reply:
@ThanGrove adding a new rule to `.gitignore` won't automatically remove any files. So since the `.gitkeep` is already tracked in git using `logs/*` is safe here. You also have the option of using `git add -f` to force adding an ignored file to version control after the fact.

Reply:
But your change still works!